components/director/internal/systemfetcher/model.go

```diff
@@ -0,0 +1,58 @@
+package systemfetcher
+
+import (
+	"encoding/json"
+
+	"github.com/tidwall/gjson"
+)
+
+var (
+	// Mappings global static configuration which is set after reading the configuration during startup, should only be used for the unmarshaling of system data
+	Mappings []TempMapping
+)
+
+type TempMapping struct {
```

Comment:
Can you rename this to `TemplateMapping`, because it otherwise sounds like a temporary struct for holding things?